### PR TITLE
Small English fixes

### PR DESCRIPTION
**Pull request by Zirak**
Changed the wording a bit, reorganised some sentences, made some waffles.

That's a lie, no waffles were made.

Also, I'm not worthy of you.

### CODE REVIEW

**Comment by Ralt** [comment]
I WANT WAFFLES NOW

**Comment by Canop** [comment]
Chuis un peu au boulot, hein...

File: src/rsc/intro.html [hunk 1]
```diff
@@ -17,11 +17,11 @@
   </a>
 </header>
 <section id="responsive" class="center">
-  <h1>Powerful free chat rooms</h1>
-  <p>An open-source chat with the features you expect and many other ones.</p>
+  <h1>Powerful, free chat rooms</h1>
+  <p>An open-source chat with features you'd expect much more.</p>
```
Comment (Ralt) [inline src/rsc/intro.html]: > with features you'd expect much more

Uh is that really correct english?

Comment (Canop) [inline src/rsc/intro.html]: Doesn't seem so. I'll keep the old wording for this line until solved.

Comment (Zirak) [inline src/rsc/intro.html]: durrr

**Pull request closed without merging**